operator-lifecycle-manager.Dockerfile

```diff
@@ -14,6 +14,8 @@ RUN mkdir -p .git/objects
 COPY . .
 RUN make build/olm bin/cpb
 
+RUN chmod +x /build/bin/collect-profiles
```

Comment:
I'm surprised that this would be necessary. What's special about how this binary is being produced?

Reply:
I was too. Now that the PR is in a working state I will attempt to remove it and diagnose why the binary could not be ran.